diff --git a/node_modules/gatsby-source-jaen/dist/source-nodes/jaen-data.js b/node_modules/gatsby-source-jaen/dist/source-nodes/jaen-data.js
index 21d71ca..10f88e3 100644
--- a/node_modules/gatsby-source-jaen/dist/source-nodes/jaen-data.js
+++ b/node_modules/gatsby-source-jaen/dist/source-nodes/jaen-data.js
@@ -95,7 +95,14 @@ var sourceNodes = function (args) { return __awaiter(void 0, void 0, void 0, fun
                 });
                 if (response) {
                     jaenData = (0, deepmerge_1.default)(jaenData, response.data, {
-                        arrayMerge: deepmerge_2.deepmergeArrayIdMerge
+                        arrayMerge: deepmerge_2.deepmergeArrayIdMerge,
+                        customMerge: function (key) {
+                            if (key === 'IMA:MdxField') {
+                                return function (target, source) {
+                                    return __assign(__assign({}, target), source);
+                                };
+                            }
+                        }
                     });
                 }
                 _a.label = 5;
diff --git a/node_modules/gatsby-source-jaen/dist/utils/deepmerge.js b/node_modules/gatsby-source-jaen/dist/utils/deepmerge.js
index 1e8da33..bbd52fd 100644
--- a/node_modules/gatsby-source-jaen/dist/utils/deepmerge.js
+++ b/node_modules/gatsby-source-jaen/dist/utils/deepmerge.js
@@ -24,6 +24,12 @@ var deepmergeArrayIdMerge = function (target, source, options) {
     if (source == null) {
         return target.slice();
     }
+    // Check if target and source are arrays of primitives
+    if (target.every(function (v) { return typeof v !== 'object'; }) &&
+        source.every(function (v) { return typeof v !== 'object'; })) {
+        // Take the source array
+        return source.slice();
+    }
     var groups = ['id', 'fieldName'];
     var _loop_1 = function (group) {
         if (target.every(function (v) { return v === null || v === void 0 ? void 0 : v[group]; }) && // Check if target elements and group property exist
